backend/audit/mixins.py

```diff
@@ -65,7 +65,8 @@ def dispatch(self, request: HttpRequest, *args: Any, **kwargs: Any) -> HttpRespo
                 eligible_users = [acc.user for acc in eligible_accesses]
 
                 raise CertificationPermissionDenied(
-                    "bad role", eligible_users=eligible_users
+                    "Bad Role. User is not the Auditee Certifying Official.",
```

Comment:
Can we change this to `Invalid role. User is not the Auditee Certifying Official.`?

Reply:
Sure thing!